modules/developers-guide/pages/metrics.adoc

```diff
@@ -0,0 +1,36 @@
+= Using metrics with Stargate
+
+Stargate is a data gateway deployed between client applications and a database.
+Metrics can be monitored using Prometheus and Grafana. A
+https://github.com/stargate/docker-images/[`docker-compose`
+script] is available to run Apache Cassandra^(TM)^, Stargate, Prometheus, and Grafana
+together to explore the metrics available.
+
+// tag::prereqsList[]
+include::partial$prereqs.adoc[]
+// end::prereqsList[]
+
+// tag::getDockerImage[]
+include::partial$docker_compose.adoc[]
+// end::getDockerImage[]
+
+== Access Prometheus
+Once all the images are running, you can access Prometheus in the browser at
+http://localhost:9090/[http://localhost:9090/].
+
+In order to access the Stargate metrics, you must modify the `/etc/prometheus/prometheus.yml`
```

Comment:
Can this file be mounted as a volume so that it works directly from `docker-compose up` rather than needing a restart?

Reply:
Probably. Want to help me with that? :-) I can figure it out, but I'm sure it will take me lots longer than you @dougwettlaufer.

Reply:
Is this what would be added to the docker-compose.yml for the prometheus image:
`
 volumes:
      - promconfig:/etc/prometheus
volumes:
  promconfig:
`

Reply:
Should just be 

```yaml
    volumes:
      - ${PWD}/prometheus.yml:/etc/prometheus/prometheus.yml
```